intl/messages/en.json

```diff
@@ -18,7 +18,12 @@
     "textDescription": "js-ipfs paves the way for the Browser implementation of the IPFS protocol. Written entirely in JavaScript, it **runs in a Browser, a Service Worker, a Web Extension and Node.js**, opening the door to a world of possibilities.",
     "currentVersion": "Current version:",
     "latestUpdate": "Latest update:",
-    "downloadsLastMonth": "Downloads last month:"
```

Comment:
Use a placeholder for the complete translation, e.g.: "Downloads last month: <%- count %>".

The same applies to everything similar, e.g.: currentVersion latestUpdate and so on.